plugin/ControllerFlickable.qml

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (C) 2019 Joona Petrell.
+ * Contact: Joona Petrell <joona.petrell@jolla.com>
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; version 2 only.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+import QtQuick 2.0
```

Comment:
Just a suggestion, but could aim to introduce new QtQuick imports as 2.6 at this point.